src/masonite/stubs/api/Resource.py

```diff
@@ -0,0 +1,14 @@
+from masonite.api.resources import Resource
+
+
+class __class__(Resource):
+    def show(self, request, __record___id):
```

Comment:
should we add a line (that the developer will complete after scaffolding ?) ?
```python
__model__ =
```

Reply:
ah no okay I got it ! The model is written in all methods, I guess it's more flexible ?
I was thinking that we would do something like this instead : 
```python
class __class__(Resource):
     __model__ = User

    def show(self, request, __record___id):
         return self.__model__.find(__record___id)

    def index(self, request):
        return self.__model__.paginate(request.input("limit", 20), request.input("page", 1))
```